docs/data-management/qaqc-criteria.md

```diff
@@ -0,0 +1,196 @@
+# QA/QC criteria
+
+The following lists the pre-defined exclusion criteria for analyses of whole-brain structural and functional connectomes.
+
+## Anatomical MRI
+
+Given our planned analysis, the T1w image will be used for the spatial alignment with the standard MNI152NLin2009cAsym template.
+In addition, surface reconstructions from the T1w image will guide the co-registration of structural and functional (BOLD) images in fMRIPrep.
+Since the latter preprocessing steps are relatively robust to structural images with mild artifacts, the exclusion criteria for unprocessed T1w images are lenient.
+However, individual T1w images may be excluded without such a decision lead to exclusion of the whole session in which it belongs.
+We annotate subjects with visible artifacts in the T1w images in order to ensure rigorous scrutinizing of spatial normalization and surface reconstruction outputs from fMRIPrep (if both modalities passed the first QC checkpoint with MRIQC).
+The explanation and the description of the flagging criteria 1 to 7 are the same as their counterpart in the previous section.
+
+## Summary
+
+#. Artifactual structures in the background
+#. Susceptibility distortion artifacts
+#. Aliasing ghosts
+#. Wrap-around that overlaps with the brain
+#. Data formatting issues
+#. Motion-related and Gibbs ringing
+#. Extreme intensity non-uniformity
+#. Eye spillover
```

Comment:
This section should be easy to read and direct. Needing a summary suggests the opposite: some metadiscourse is necessary to be able to delve into the section. We should avoid this.